test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -198,6 +199,64 @@ public async Task DotnetRestore_OneLinePerRestore()
             }
         }
 
+        [PlatformFact(Platform.Windows)]
+        public async Task DotnetRestore_ProjectMovedDoesNotRunRestore()
+        {
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                var testDirectory = pathContext.SolutionRoot;
+                var pkgX = new SimpleTestPackageContext("x", "1.0.0");
+                pkgX.Files.Clear();
+                pkgX.AddFile($"lib/netstandard2.0/x.dll", "netstandard2.0");
+
+                await SimpleTestPackageUtility.CreatePackagesAsync(pathContext.PackageSource, pkgX);
+
+                var projectName = "ClassLibrary1";
+                var projectDirectory = Path.Combine(testDirectory, projectName);
+                var movedDirectory = Path.Combine(testDirectory, projectName + "-new");
+
+                var projectFile1 = Path.Combine(projectDirectory, $"{projectName}.csproj");
+                var movedProjectFile = Path.Combine(movedDirectory, $"{projectName}.csproj");
+
+                _msbuildFixture.CreateDotnetNewProject(testDirectory, projectName, " classlib");
+
+                using (var stream = File.Open(projectFile1, FileMode.Open, FileAccess.ReadWrite))
+                {
+                    var xml = XDocument.Load(stream);
+
+                    var attributes = new Dictionary<string, string>() { { "Version", "1.0.0" } };
+
+                    ProjectFileUtils.AddItem(
+                        xml,
+                        "PackageReference",
+                        "x",
+                        (NuGetFramework)null,
+                        new Dictionary<string, string>(),
+                        attributes);
+
+                    ProjectFileUtils.WriteXmlToFile(xml, stream);
+                }
+
+
+                // Act
+                var result = _msbuildFixture.RunDotnet(projectDirectory, $"build {projectFile1} {$"--source \"{pathContext.PackageSource}\""}", ignoreExitCode: true);
+
+
+                // Assert
+                Assert.True(result.Item1 == 0, result.AllOutput);
```

Comment:
Use `result.ExitCode` instead of `result.Item1` here and below.  The former is more understandable than the latter.  (Really, we should eliminate `Item1`, `Item2`, and `Item3` but not in this PR.